backend/src/controllers/OrderController.ts

```diff
@@ -48,14 +51,18 @@ const createCheckoutSession = async (req: Request, res: Response) => {
     );
 
     if (!session.url) {
-      return res.status(500).send('Error creating checkout session');
+      throw new AppError('Error creating checkout session', 500);
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider more specific error message</b></div>
<div id="fix">

Consider using a more specific error message that provides details about the nature of the error. This could help with debugging and troubleshooting.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -54 +54 @@
 -      throw new AppError('Error creating checkout session', 500);
 +      throw new AppError('Failed to create checkout session: No URL returned', 500);
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/omkardongre/Food-Ordering-App/pull/47#issuecomment-2439370000>#eef1bf</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged